docs/celestia-architecture/adr-004-mvp-light-client.md

```diff
@@ -259,7 +259,7 @@ A DAS light client will have to request the `DASLightBlock` for this as per abov
 
 Ideally, we add the DAS light client to the existing e2e tests.
 It might be worth to catch up with some relevant changes from tendermint upstream.
-In particular, [tendermint/tendermint#6196](https://github.com/tendermint/tendermint/pull/6196) and previous changes that it depends on.
+In particular, [tendermint/tendermint#6196](https://github.com/cometbft/cometbft/pull/6196) and previous changes that it depends on.
```

Comment:
explicit links on the other hand (like http:// whatever) we might actually want to fix

that or disable the dead link linter